.github/workflows/ci-common.yml

```
@@ -0,0 +1,40 @@
+name: CI common
```

since this is the only CI action, i think we can shorten this as well as the filename as `ci.yml`

```suggestion
name: CI
```